docs/developer/developer-handbook.md

```diff
@@ -0,0 +1,93 @@
+# EDC Developer Handbook
+
+<!-- TOC -->
+* [EDC Developer Handbook](#edc-developer-handbook)
+  * [Introduction](#introduction)
+  * [Building a distribution](#building-a-distribution)
+    * [Seed some data](#seed-some-data)
+    * [Perform a data transfer](#perform-a-data-transfer)
+  * [The control plane - core concepts](#the-control-plane---core-concepts)
+    * [API objects in detail](#api-objects-in-detail)
+    * [EDC state machines](#edc-state-machines)
+    * [The extension model](#the-extension-model)
+    * [EDC dependency injection](#edc-dependency-injection)
+    * [Policy scopes and evaluation](#policy-scopes-and-evaluation)
+  * [The data plane](#the-data-plane)
+    * [Data plane selectors](#data-plane-selectors)
+    * [Writing a DataSink and DataSource extension](#writing-a-datasink-and-datasource-extension)
+    * [The Control API](#the-control-api)
+  * [Advanced concepts](#advanced-concepts)
```

Comment:
we have chapters for the connector concepts but not for IH, RS and FCC

Reply:
that's on purpose - according to the outline defined [here](https://github.com/orgs/eclipse-edc/discussions/61#discussioncomment-6656353) each component should discuss their internals in their own README, to which we will link in the two-line overview.

Reply:
ok, but here, in the general docs we're talking about the connector functionality (control plane + data plane = connector). I agree that detailed stuff should stay in the related repository, but an "8 miles high" view of the whole set of components should stay here, like, what's an identity hub and how it interacts with a connector for example

Reply:
yep - i thought that was the two-line thing